tests/test-definition.yaml

```diff
@@ -16,24 +16,24 @@ dimensions:
       - "12.5.6"
   - name: hive
     values:
-      - 3.1.3
-      - 4.0.0
-      - 4.0.1
+      #- 3.1.3,local-hive:3.1.3
+      #- 4.0.0,local-hive:4.0.0
+      - 4.0.1,local-hive:4.0.1
```

Comment:
Accidental commits?

Reply:
Oops, yes

Reply:
Fixed in 9edfbd46a11064638cc3e0e3b99af1d1bdbd1c70